tools.mk

```diff
@@ -27,6 +27,13 @@
 REBAR ?= ./rebar
 REVISION ?= $(shell git rev-parse --short HEAD)
 PROJECT ?= $(shell basename `find src -name "*.app.src"` .app.src)
+REBAR_DEPS_DIR ?= deps
+
+ifeq ($(shell uname -s),Linux)
+ESED ?= sed -r
+else
+ESED ?= sed -E
```

Comment:
`-E` is either POSIX or de-facto Unix standard, and most Linux distros seem to support it, but their docs all say `-r` is the switch to use.